src/components/Tickets/BookingPage.js

```diff
@@ -0,0 +1,148 @@
+  const [bookingHomeVisible, setBookingHomeVisible] = useState(true);
+
+  const seats = {
+    row: ["A", "B", "C", "D"],
+    data: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+  };
+  let location = useLocation();
+
+  const handleSeatClick = (id) => {
+    if (selectedSeats.includes(id)) {
+      let newSelectedSeats = selectedSeats.filter((seat) => {
+        return id !== seat;
+      });
+      setSelectedSeats(newSelectedSeats);
+    } else if (selectedSeats.length < 10) {
+      setSelectedSeats([...selectedSeats, id]);
+    }
+  };
+
+  const confirmBooking = () => {
+    setConfirmedSeats((prevSeats) => [...prevSeats, ...selectedSeats]);
+    localStorage.setItem(
+      location.state.movie.id,
+      [...confirmedSeats, ...selectedSeats].join(",")
+    );
+    if (selectedSeats.length > 0) {
+      setModalVisible(true);
+      setBookingHomeVisible(false);
+    } else {
+      alert("select minimum 1 seat");
+    }
+  };
+
+  const closeModal = () => {
+    setModalVisible(false);
+    setBookingHomeVisible(true);
+    setSelectedSeats([]);
+  };
+
+  useEffect(() => {
+    if (localStorage.getItem(location.state.movie.id)) {
+      setConfirmedSeats(
+        localStorage.getItem(location.state.movie.id).split(",")
+      );
+    }
+  }, [location]);
+
+  return (
+    <>
+      <GlobalStyle />
+      <BookingPageStyle bookingHomeVisible={bookingHomeVisible}>
+        <BookingPageFlex>
+          <ScreenStyle>
+            <img src={screen} alt="Screen this side" />
+          </ScreenStyle>
+
+          <SeatNumberStyle>
+            {seats.data.map((num) => {
+              return (
+                <SeatStyle>
+                  <h1>{num}</h1>
+                </SeatStyle>
+              );
+            })}
+          </SeatNumberStyle>
+
+          <AllSeatsStyle>
+            {seats.row.map((item) => {
+              return (
+                <RowSeatStyle>
+                  <LetterSeatStyle>
+                    <h1>{item}</h1>
+                  </LetterSeatStyle>
+                  {seats.data.map((seatNumber) => {
+                    return (
+                      <>
+                        {(() => {
+                          switch (true) {
+                            case confirmedSeats.includes(item + seatNumber):
+                              return <Seat colorN="#626262" />;
+                            case selectedSeats.includes(item + seatNumber):
+                              return (
+                                <Seat
+                                  colorN="#724FD8"
+                                  onClick={() => {
+                                    handleSeatClick(item + seatNumber);
+                                  }}
+                                />
+                              );
+                            default:
+                              return (
+                                <Seat
+                                  colorN="#DADADA"
```

Comment:
```suggestion
                                  color="#DADADA"
```

or

```suggestion
                                  colorName="#DADADA"
```

Please choose meaningful name.